src/app/public/modules/tabs/tabset.component.html

```diff
@@ -63,9 +62,8 @@
       class="sky-btn sky-btn-icon sky-tabset-btn-open sky-margin-inline-compact"
       type="button"
       [attr.aria-label]="'skyux_tab_open' | skyLibResources"
-      [ngClass]="{
-        'sky-margin-inline-sm':
-          (themeSvc?.settingsChange | async)?.currentSettings?.theme?.name === 'modern'
```

Comment:
You can remove `@Optional() public themeSvc?: SkyThemeService` from the constructor in the `ts` file.